M2/Macaulay2/packages/GKMManifolds.m2

```diff
@@ -0,0 +1,1462 @@
+-*-------------------------------------------------------------------------------------------
+   Copyright 2020 Christopher Eur and Ritvik Ramkumar.
+
+   You may redistribute this file under the terms of the GNU General Public
+   License as published by the Free Software Foundation, either version 2 of
+   the License, or any later version.
+-------------------------------------------------------------------------------------------*-
+
+
+
+newPackage("GKMManifolds",
+	Version => "0.1",
+	Date => "August 1, 2020",
+	Authors => {
+	    {Name => "Chris Eur",
+       	     Email => "chriseur@stanford.edu",
+       	     HomePage => "https://web.stanford.edu/~chriseur"},
+	    {Name => "Ritvik Ramkumar",
+	     Email => "ritvik@math.berkeley.edu",
+	     HomePage => "https://math.berkeley.edu/~ritvik"}
+	    },
+	Headline => "computations with GKM manifolds and moment graphs",
+	HomePage => "https://github.com/chrisweur/GKMManifolds",
+	PackageExports => {"Graphs", "Matroids", "NormalToricVarieties"},
+	AuxiliaryFiles => true,
+	--Keywords => {"Equivariant Cohomology", "Toric Geometry", "Flag Varieties", "Matroids"},
+	DebuggingMode => false
+)
+export {
+	"TVariety",
+	"tVariety",
+    	"TKClass",
+	"tKClass",
+	"ampleTKClass",
+	"trivialTKClass",
+	"TMap",
+	"tMap",
+	"pushforward",	
+	"diagonalTMap",
+	"FlagMatroid",
+	"flagMatroid",
+	--"tFlagVariety",
+	"tFlagMap",
+	"kTutte",
+	"hilb",
+	"tvar",
+	"charts",
+	"points",
+	"ptsMap",
+	"charRing",
+	"HTpt",
+	"tProjectiveSpace",
+	"MomentGraph",
+	"momentGraph",
+	"constituents",
+	"makeCharRing",
+	--"makeHTpt",
+	"tGeneralizedFlagVariety",
+	"lieType",
+	"cellOrder",
+	"bruhatOrder",
+	"tGeneralizedSchubertVariety",
+	"tChi",
+	"tOrbitClosure",
+	"toFraction",
+	"affineToricRing",
+	"setIndicator",
+	--"unastrsk",
+	--"toCharRing",
+	--"tHilbNumer",
+	"RREFMethod",
+	"latticePts"
+}
```

Comment:
Can you come up with less cryptic names for these identifiers?  The especially cryptic ones are "HTpt", "tvar", "hilb", "tMap", "TKclass", etc.  Even "latticePts" could be expanded to "latticePoints".

By the way, if "T" in some of the identifiers refers to your favorite symbol for a torus, then what about users whose favorite name is "S"?